api/src/main/java/io/opentelemetry/api/trace/PropagatedSpan.java

```diff
@@ -61,7 +62,7 @@ public Span addEvent(String name) {
   }
 
   @Override
-  public Span addEvent(String name, long timestamp) {
+  public Span addEvent(String name, long timestamp, TimeUnit unit) {
```

Comment:
Why not using Java.time.Instant? We are java8 now.

Reply:
I think I found myself the reason, is android version, the `java.time.Instant` was added in Android API level 26, and we try to be API level 24 compatible.

Reply:
Adding overloads with Instant sounds good too just not for now.